lizrd/scripts/grid_utils.py

```diff
@@ -27,6 +27,28 @@ def get_machine_backend() -> MachineBackend:
         return MachineBackend.LOCAL
 
 
+def get_common_directory(runner: MachineBackend) -> str:
+    if runner == MachineBackend.ATHENA:
+        return "/net/pr2/projects/plgrid/plggllmeffi"
+    elif runner == MachineBackend.IDEAS:
+        return "/raid/NFS_SHARE/llm-random"
+    elif runner == MachineBackend.ENTROPY_GPU:
+        return "/common/llm-random"
+    else:
+        return "~"
+
+
+def get_cache_path(runner: MachineBackend) -> str:
+    common_dir = get_common_directory(runner)
+    return f"{common_dir}/.cache"
```

Comment:
Default HF folder would be ~/.cache/huggingface/datasets (at least locally). Maybe let's somehow keep that?

Reply:
Maybe f"{os.getenv('HOME')}/.cache/huggingface/datasets"?